src/form/form-group/form-group.html

```diff
@@ -1,9 +1,9 @@
-<div class="form-group" ng-class="{ 'has-error' : hasErrors() }">
-  <label for="{{ pfField }}" class="control-label {{ pfLabelClass }}">{{ pfLabel }}</label>
-  <div class="{{ pfInputClass }}">
+<div class="form-group" ng-class="{ 'has-error' : $ctrl.hasErrors() }">
+  <label for="{{ $ctrl.pfField }}" class="control-label {{ $ctrl.pfLabelClass }}">{{ $ctrl.pfLabel }}</label>
```

Comment:
Should this be updated to reflect the discussed changes to `pf-form-group` (i.e. to remove the horizontal form specific classes)?